### PR TITLE
Remove auto links

### PR DESCRIPTION
**Pull request by teoli2003**
Removed a few auto links (these _See also_ should be curated/modified/deleted at some point, but this is outside the scope of this PR.

I also did a drive-by fix to remove a red link (the glossary entry didn't exist).

### CODE REVIEW

File: files/en-us/web/css/used_value/index.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ CSS 2.0 defined only _computed value_ as the last step in a property's calculati
   - Values
     - [Initial values](/en-US/docs/Web/CSS/initial_value)
     - [Computed values](/en-US/docs/Web/CSS/computed_value)
-    - [Used values](/en-US/docs/Web/CSS/used_value)
+    - **Used values**](/en-US/docs/Web/CSS/used_value)\*\*
```
Comment (estelle) [inline files/en-us/web/css/used_value/index.md:116]:
```suggestion
    - **Used values**
```

File: files/en-us/web/css/value_definition_syntax/index.md [hunk 2]
```diff
@@ -425,6 +425,6 @@ But not:
     - [Computed values](/en-US/docs/Web/CSS/computed_value)
     - [Used values](/en-US/docs/Web/CSS/used_value)
     - [Actual values](/en-US/docs/Web/CSS/actual_value)
-  - [Value definition syntax](/en-US/docs/Web/CSS/Value_definition_syntax)
+  - **Value definition syntax**](/en-US/docs/Web/CSS/Value_definition_syntax)\*\*
```
Comment (estelle) [inline files/en-us/web/css/value_definition_syntax/index.md:428]:
```suggestion
  - **Value definition syntax**
```

File: files/en-us/web/css/visual_formatting_model/index.md [hunk 3]
```diff
@@ -145,7 +145,7 @@ A block box is a block-level box that is also a block container. As described in
   - [Inheritance](/en-US/docs/Web/CSS/inheritance)
   - [Box model](/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model)
   - [Layout modes](/en-US/docs/Web/CSS/Layout_mode)
-  - [Visual formatting models](/en-US/docs/Web/CSS/Visual_formatting_model)
+  - **Visual formatting models**](/en-US/docs/Web/CSS/Visual_formatting_model)\*\*
```
Comment (estelle) [inline files/en-us/web/css/visual_formatting_model/index.md:148]:
```suggestion
  - **Visual formatting models**

**Review by estelle — COMMENTED** [review]
I think there were a few oopsies.

**Comment by teoli2003** [comment]
Ooops!

**Review by estelle — APPROVED** [review]

**Pull request merged**